### PR TITLE
Examples: Clean up.

### PR DESCRIPTION
**Pull request by Mugen87**
Related issue: -

**Description**

Removed the animation loop in `webgl_loader_ifc` and structured the code similar to the other examples.

It also fixes the path to the core library in `IFCLoader`.

### CODE REVIEW

**Comment by Mugen87** [comment]
https://raw.githack.com/Mugen87/three.js/dev46/examples/webgl_loader_ifc.html

**Comment by Mugen87** [comment]
https://raw.githack.com/Mugen87/three.js/8a20cb59e445dca10a419d42a4395103a666d543/examples/webgl_loader_ifc.html

File: examples/jsm/loaders/IFCLoader.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ import {
 	InterleavedBuffer,
 	InterleavedBufferAttribute,
 	BufferAttribute,
-} from '../../../../build/three.module.js';
+} from '../../../build/three.module.js';
```
Comment (mrdoob) [inline examples/jsm/loaders/IFCLoader.js:17]: Ops! Can't wait for Import Maps...

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**